src/main/java/com/odysseusinc/arachne/portal/service/impl/BaseGlobalSearchServiceImpl.java

```diff
@@ -118,7 +118,7 @@ private String[] calculateCollections(final SearchGlobalDTO searchDTO) {
 
     private String buildSearchString(final String query, final Long userId, final String tenantId) {
 
-
+        // TODO do we need to change here, studies -> studies_data?
```

Comment:
Any errors?

Reply:
The answer is yes, we need to change eventually.
Such a change might require an update to  indexing configuration, so I have refrained from performing it.
System-to-system integration is not using this service, so session multi-tenancy should not provoke any errors here.

Reply:
A separate task has been created